### ISSUE TITLE
registerRenderer is mentioned in some places

### ISSUE THREAD

**Issue by squadette** [shakacode/react_on_rails#1029]
Four files still mention the absent function:

````
sq@MacBook-Pro ~/src/react_on_rails$ git grep registerRenderer
spec/dummy/app/views/pages/client_side_manual_render.html.erb:  ReactOnRails.registerRenderer({
spec/dummy/app/views/pages/deferred_render_with_server_rendering.html.erb:  ReactOnRails.registerRenderer({
spec/dummy_no_webpacker/app/views/pages/client_side_manual_render.html.erb:  ReactOnRails.registerRenderer({
spec/dummy_no_webpacker/app/views/pages/deferred_render_with_server_rendering.html.erb:  ReactOnRails.registerRenderer({
````

Looks like it was forgotten while working on https://github.com/shakacode/react_on_rails/pull/581

**Comment by justin808** [comment]
Good catch. Can you submit a PR?

**Comment by squadette** [comment]
I'm not sure because I could fully understand how it works. :)

**Comment by justin808** [comment]
Fixed in #1059
